benchmarks/reframe_config.py

```diff
@@ -157,6 +157,39 @@ def spack_root_to_path():
                 },
             ]
         },  # end CSD3 CentOS 7
+        {
+            # https://www.rc.ucl.ac.uk/docs/Clusters/Kathleen/#node-types
+            'name': 'kathleen',
+            'descr': 'Kathleen',
+            'hostnames': ['login[0-9]+.kathleen.ucl.ac.uk'],
+            'max_local_jobs': 1,
+            'partitions': [
+                {
+                    'name': 'compute-node',
+                    'descr': 'Kathleen compute nodes',
+                    'scheduler': 'sge',
+                    'launcher': 'mpirun',
+                    'environs': ['default'],
+                    'max_jobs': 36,
+                    'processor': {
+                        'num_cpus': 80,
+                        'num_cpus_per_core': 2,
+                        'num_sockets': 2,
+                        'num_cpus_per_socket': 20,
```

Comment:
I feel like this should represent the physical, not virtual cores. If a benchmark wants to run on a "full node" it would grab the `num_cpus` number and request that many cores per node. I assume with `-l threads=2` it will get rejected by the scheduler? I guess I need to go and test this. But since we are disabling hyperthreading by default I think the processor config should reflect this.

Reply:
> But since we are disabling hyperthreading by default 

What do you mean exactly?

Reply:
> Are you suggesting we should set `num_cpus_per_core=1` and then `num_cpus=40`?

Yes, I think we should. We do the same e.g. on Archer2 where we set 

https://github.com/ukri-excalibur/excalibur-tests/blob/3a9f51fe0b275c5ba75b4d1ece92ceef61e4fb7c/benchmarks/reframe_config.py#L60-L64

even though the number of CPU(s) reported by `lscpu` is 256 with 2 Threads per core.

At least we should be consistent on how we set this across our systems.